### PR TITLE
To address issue #42, added a parameter to the all_mismatch method to control the addition of matching columns to the output.

### PR DESCRIPTION
**Pull request by azadekhalaj**

### CODE REVIEW

**Comment by fdosani** [comment]
@azadekhalaj can you make sure to sign the CLA please! I'll take a review shortly here.

**Comment by azadekhalaj** [comment]
> @azadekhalaj can you make sure to sign the CLA please! I'll take a review shortly here.

Sure. I think I will need to receive an approval from Legal before singing the CLA.

File: datacompy/core.py [hunk 1]
```diff
@@ -498,8 +505,29 @@ def all_mismatch(self):
...
+
+                if not ignore_matching_cols or (
+                    ignore_matching_cols and not col_comparison.all()
+                ):
+                    LOG.debug("Adding column {} to the result.".format(orig_col_name))
+                    match_list.append(col)
+                    return_list.extend([orig_col_name + "_df1", orig_col_name + "_df2"])
+                elif ignore_matching_cols:
+                    LOG.debug(
+                        "Columns {} are equal in df1 and df2 and not adding to the result.".format(
```
Comment (fdosani) [inline datacompy/core.py:527]:
```suggestion
                        "Column {} is equal in df1 and df2. It will not be added to the result.".format(
```

**Review by fdosani — COMMENTED** [review]
@azadekhalaj Looks great, thanks for contributing!
One minor change in terms of the debug log wording if you are OK with the suggested changes.

File: datacompy/core.py (hunk 1 above)
Comment (azadekhalaj) [reply datacompy/core.py]: Definitely. I updated the log message.

**Review by fdosani — APPROVED** [review]
LGTM

**Pull request merged**